### ISSUE TITLE
Use Goroutines and channels to speed up the process

### ISSUE THREAD

**Issue by gadelkareem** [ikeikeikeike/go-sitemap-generator#18]
It would be great if we can create sitemaps through Goroutines and use a channel per 50000 URLs for ex. Is it possible to add this as feature request?

**Comment by ikeikeikeike** [comment]
Yes, It's seriously will be great though, I'm afreid that I cannot impliment due to too busy recently.
